src/Geometry/conversions.jl

```diff
@@ -141,21 +141,18 @@ LocalVector(u::ContravariantVector{<:Any, (3,)}, ::LocalGeometry{(1, 2)}) =
     CovariantVector(u, local_geometry).u₃
 
 @inline contravariant1(u::AxisVector, local_geometry::LocalGeometry) =
-    transform(Contravariant123Axis(), u, local_geometry).u¹
+    @inbounds project(Contravariant1Axis(), u, local_geometry)[1]
```

Comment:
Wait, why was this changed from `Contravariant123Axis` to `Contravariant1Axis`? I assume this is equivalent? cc @simonbyrne

Reply:
We only care about the 1st component